### ISSUE TITLE
roxygen2 version 5.0.0 breaks rstanarm build

### ISSUE THREAD

**Issue by bgoodri** [stan-dev/rstanarm#31]
The error is
```
Quitting from lines 107-113 (aov.Rmd) 
Error: processing vignette 'aov.Rmd' failed with diagnostics:
'stan_lm' is not an exported object from 'namespace:rstanarm'
Execution halted
```
which appears to be consistent with the new auto-generated NAMESPACE, which may have to do with these warnings
```
roxygen2::roxygenize(clean = TRUE)
...
There were 14 warnings (use warnings() to see them)
> warnings()
Warning messages:
1: @export [posterior_predict.R#12]: may only span a single line
2: @export [predict.R#4]: may only span a single line
3: @export [rstanarm-package.R#14]: may only span a single line
4: @export [stan_glm.R#23]: may only span a single line
5: @export [stan_glm.fit.R#26]: may only span a single line
6: @export [stan_glmer.R#7]: may only span a single line
7: @export [stan_glmer.R#113]: may only span a single line
8: @export [stan_lm.R#25]: may only span a single line
9: @export [stan_polr.R#46]: may only span a single line
10: @export [stanreg-methods.R#99]: may only span a single line
11: @export [stanreg-methods.R#227]: may only span a single line
12: @export [stanreg-methods.R#239]: may only span a single line
13: @export [stanreg-methods.R#249]: may only span a single line
14: @export [summary.R#8]: may only span a single line
```
```

**Comment by bgoodri** [comment]
Apparently you are not allowed to have a blank line after `@export` anymore. Fix on the way.

**Comment by bgoodri** [comment]
Fixed by 1313d62

**Comment by jgabry** [comment]
Thanks for fixing that.
